pipeline-modules/app-service/aws/semgrep.yaml

```diff
@@ -0,0 +1,67 @@
+provisioner: aws
+name: semgrep
+version: 1
+revision: 1
+displayName: semgrep
+description: Run semgrep scans
+target: ""
+category: scan
+keywords:
+  - scan
+  - sast
+author: CloudCover
+meta: {}
+inputs:
+  properties:
+    working_dir:
+      title: Working directory
+      description: The current working directory to execute the bash script from
+      type: string
+      default: repo
+    allow_critical:
+      title: Allow Critical Vulnerabilities
+      description: Allows builds with critical vulnerabilites
+      type: boolean
+      default: false
+    allow_error:
+      title: Allow Error Vulnerabilities
+      description: Allows Error vulnerabilites
+      type: boolean
+      default: false
+template: |
+  version: 0.2
+  env:
+    shell: bash
```

Comment:
I believe this is already set in the pipeline-config. so setting it again may cause conflict in module stitching logic